mypy/util.py

```diff
@@ -384,13 +467,21 @@ def style(self, text: str, color: Literal['red', 'green', 'blue', 'yellow', 'non
             start = ''
         if underline:
             start += self.UNDER
+        if dim:
+            start += self.DIM
         return start + self.colors[color] + text + self.NORMAL
 
     def colorize(self, error: str) -> str:
         """Colorize an output line by highlighting the status and error code."""
         if ': error:' in error:
             loc, msg = error.split('error:', maxsplit=1)
-            if not self.show_error_codes:
+            if self.show_source_code:
+                # Improve readability by wrapping lines when showing source code.
```

Comment:
It's can be surprising that showing source code also affects how long messages are wrapped. Maybe the option should be named `--pretty` or something, as it controls a few things.